assets/app/features/upgrade/controllers/upgrade-database-configuration.controller.js

```diff
@@ -41,7 +46,13 @@
         vm.createServer = createServer;
         vm.connectServer = connectServer;
 
+        activate();
+
         // functions
+        function activate() {
+            upgradeStatusFactory.syncStatusFlags(UPGRADE_STEPS.database, vm,
+                undefined, upgradeStepsFactory.setCurrentStepCompleted);
```

Comment:
ah shit, this is done in the syncStatusFlags `flagsObject.running`

I didnt see that, deleting the comments!